lib/shared/units.js

```diff
@@ -91,3 +100,27 @@ exports.bytesToClosestUnit = (bytes) => {
 exports.daysToMilliseconds = (days) => {
   return days * MILLISECONDS_IN_A_DAY
 }
+
+/**
+ * @summary Convert milliseconds to seconds
+ * @function
+ * @public
+ *
+ * @param {Number} time - millisecond time
+ * @returns {Number} seconds time fractional
+ *
+ * @example
+ * const DELAY = 10000
+ * setTimeout(() => {
+ *   console.log(`${units.millisecondsToSeconds(DELAY)} seconds passed.`)
+ * }, DELAY)
+ */
+exports.millisecondsToSeconds = (time) => {
+  if (!Number.isFinite(time)) {
+    throw errors.createError({
+      title: `Not a finite number: ${time}`
+    })
+  }
+
+  return time / exports.MILLISECONDS_IN_A_SECOND
```

Comment:
Purely out of curiosity (I'm not querying what you've done, I'm just trying to learn), why does this use `exports.MILLISECONDS_IN_A_SECOND` but inside `daysToMilliseconds` there's no `exports.` on the front of `MILLISECONDS_IN_A_DAY` ?

Reply:
We only export `MILLISECONDS_IN_A_SECOND` because we use it later in the `60 * MILLISECONDS_IN_A_SECOND` expression. If `MILLISECONDS_IN_A_DAY` becomes needed elsewhere we could also export that

Reply:
Ahhh, thanks! I hadn't noticed that `MILLISECONDS_IN_A_DAY` was a 'private' constant and not exported :+1: